source/install/troubleshooting.rst

```diff
@@ -6,44 +6,34 @@ Troubleshooting
 Important notes
 ---------------
 
-**DO NOT manipulate the Mattermost database**
+Database
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  In particular, DO NOT delete data from the database, as Mattermost is
-   designed to stop working if data integrity has been compromised. The
-   system is designed to archive content continously and generally
-   assumes data is never deleted.
+- DO NOT manipulate the database directly. Mattermost is designed to stop working if data integrity is compromised. 
+- Any manipulation of the database must be done using the built in command-line tools
 
 Troubleshooting Basics
 ----------------------
 
-If you're new to Mattermost or troubleshooting consider the following
-steps:
+If you're new to Mattermost or troubleshooting consider the following steps:
 
-1. Start simple
-   - Use the step-by-step install guides before trying variations.
+- Start simple with the step-by-step install guides for your operating system
 
-2. Check the logs for errors
-   - In the directory where you installed Mattermost look in the ``mattermost.log`` file for error messages. Also check NGNIX error logs.
+- Check the logs (``mattermost.log`` and NGNIX logs) for errors
 
-3. Web search the error message
-   - If you have an error message you don't understand, search the web for the message to try and find an existing article.
+- Search error messages, existing solutions can often work
```

Comment:
Please specify where to search, many Mattermost users don't have English as a first language, so we need to be extra specific